agent/js/package.json

```diff
@@ -29,11 +39,28 @@
   },
   "exports": {
     "require": "./dist/index.js",
-    "default": "./dist/index.modern.js",
     "types": "./dist/index.d.ts"
   },
   "files": [
-    "dist"
+    "dist",
+    "README.md",
+    "package.json",
+    "package-lock.json"
```

Comment:
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files

`readme.md` and `package.json` are always included,  but let's keep them here, it's better to be explicit. `package-lock.json` can't be added, so remove it, to prevent some potential confusion